Documentation/admin/logging-destination.md

```diff
@@ -13,6 +13,8 @@ There are currently 4 prebuilt Debian based Docker images in the [quay.io/coreos
 
 **Note**: there are Alpine based images which are automatically published along side the Debian images, but they cannot be used in conjunction with the systemd input plugin, because Alpine has no `libsystemd` package available.
 
+The source of these docker images can be found at [coreos/fluentd-kubernetes-daemonset][github-coreos-fluentd] on Github.
```

Comment:
Can you add a note here to also mention they should use the `tectonic_logging` branch (sometimes navigating github can lose the branch your own).

Reply:
"For source information, use the tectonic_logging branch found at https://github.com/coreos/fluentd-kubernetes-daemonset/tree/tectonic_logging on GitHub."

@chancez will using the branch be a permanent requirement? or will this go pure upstream at some point?

Reply:
It won't be upstreamed, these changes are Tectonic specific.